README.md

```diff
@@ -499,6 +485,73 @@ Create a `SolidResource` as a child of this container using a UUID as the name w
 #### `deleteContents()`
 Recursively delete the contents of this container but not the container itself.
 
+### AccessDatasetBuilder
+This universal access dataset builder can be used to build up a set of rules which are then built into a WAC or ACP
+specific [AccessDataset](#accessdataset). Many of the methods below take an access parameter which is a list of access
+modes to be granted. This list can contain any of `read`, `write`, `append`, `control`, `controlRead`, `controlWrite` or
+an IRI representing an access mode not known to the test harness. The known modes are translated to the appropriate IRI
+for WAC or ACP. There are special conditions relating to the `control` modes. In WAC you can only set `control` however
+in ACP it is possible to add `read` and/or `write` access to the ACL resource. When you use the mode `control` is used
+for ACP it is translated to `read` and `write` but it is possible to set only one of these using the `controlRead` and
+`controlWrite` variants.
+
+For example:
+```js
+const access = testContainer.getAccessDatasetBuilder(webIds.alice)
+        .setAgentAccess(testContainer.getUrl(), webIds.bob, ['write'])
+        .setInheritableAgentAccess(testContainer.getUrl(), webIds.bob, ['append', 'write', 'control'])
+        .build();
+testContainer.setAccessDataset(access);
```

Comment:
My feeling when I see this, is that it is too much of a mouthful, it is wet (as in not DRY ;-) ), as there are five lines of code that is basically everywhere. Also, there are really just 5 parameters, the WebIDs and the access modes. In many cases, you wouldn't have inheritable access, and the WebIDs would usually be the same, so it would be much nicer if the code had reasonable defaults that you could override if needed. 

Would it be possible to do something that said just
```gherkin
When DirectMode = ['read','write']
```
for example? So, you wouldn't need to write JS unless you had to use non-defaults?